src/domain/community/community/community.resolver.mutations.ts

```diff
@@ -382,40 +393,60 @@ export class CommunityResolverMutations {
       `create invitation community: ${community.id}`
     );
 
-    const invitations: IInvitation[] = [];
+    return Promise.all(
```

Comment:
I believe some handling is required here. I am not sure what will happen if one invitation fails.